src/main/java/uk/gov/hmcts/reform/pip/account/management/model/AuditAction.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.pip.account.management.model;
+
+/**
+ * Enum that contains the different actions that are audited.
+ */
+public enum AuditAction {
```

Comment:
Could we put this in the pip data models repo? Only really used in one backend, but just in case in the future

Reply:
Done, build will fail until https://github.com/hmcts/pip-data-models/pull/14 is merged